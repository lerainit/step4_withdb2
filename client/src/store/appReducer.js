import { combineReducers } from "redux";
import cardIndexReducer from "./cardId/reducer";
import modalReducer from './modal/reducer'
import productsReducer from "./products/reducer";
import likesReducer from "./likes/reducer";
import BackgroundReducer from "./cardBackground/reducer";
import postsReducer from "./posts/reducer";
import UsersReducer from "./users/reducer";
import subscriberReducer from "./subscribers/reducer";
import userIndexReducer from "./userIndex/reducer";
import commentsReducer from "./comments/reducer";
import FollowerReducer from "./followers/reducer";


const appReducer = combineReducers({
   modal: modalReducer,
   cardIndex: cardIndexReducer,
   products: productsReducer,
   counter: likesReducer,
   background: BackgroundReducer,
   posts: postsReducer,
   users: UsersReducer,
   subscribers: subscriberReducer,
   userIndex: userIndexReducer,
   comments: commentsReducer,
   followers:FollowerReducer

})
export default appReducer