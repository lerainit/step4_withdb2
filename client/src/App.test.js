import { render, screen } from '@testing-library/react';
import App from './App';

import { Provider } from "react-redux";
import store from './store'

const Component =() =>{
    return (
    <Provider store ={store}>
        <App/>
    </Provider>
)}
describe('App renders',()=>{

test('should App match snapshot',()=>{
const {asFragment } = render(<Component/>)
expect(asFragment()).toMatchSnapshot()

})

})