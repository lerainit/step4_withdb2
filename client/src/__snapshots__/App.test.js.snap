// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders should App match snapshot 1`] = `
<DocumentFragment>
  <h1>
    LOADING
  </h1>
</DocumentFragment>
`;
